client/src/components/ClassForm.jsx

```diff
@@ -0,0 +1,84 @@
+import React from 'react';
+import { Modal, Form } from 'semantic-ui-react';
+
+import { putUserClass } from '../utils/apiWrapper';
+import 'semantic-ui-css/semantic.min.css';
+import '../css/ClassForm.scss';
+
+function ClassForm(props) {
+  const { button, user, setClasses } = props;
+  const [open, setOpen] = React.useState(false);
+  const [courseCode, setCourseCode] = React.useState('');
+  const [courseNumber, setCourseNumber] = React.useState('');
+  const [courseSuffix, setCourseSuffix] = React.useState('');
+
+  const handleSubmit = async () => {
+    console.log(courseCode, courseNumber, courseSuffix);
+    const classComplete = courseCode + courseNumber + courseSuffix;
+    if (courseCode === '' || courseNumber === '') {
+      return;
+    }
+    const updatedUser = {
+      memberDbId: user.memberDbId,
+      classes: [...user.classes, classComplete],
```

Comment:
Maybe change the name of this to `newClass` or something more descriptive. `classComplete` is a bit confusing on line 23